src/main/java/com/hurricane/mindmap/router/TaskRouter.java

```diff
@@ -0,0 +1,23 @@
+package com.hurricane.mindmap.router;
+
+import com.hurricane.mindmap.handler.TaskHandler;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.server.RouterFunction;
+import org.springframework.web.reactive.function.server.RouterFunctions;
+import org.springframework.web.reactive.function.server.ServerResponse;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.web.reactive.function.server.RequestPredicates.*;
+
+@Configuration
+public class TaskRouter {
+
+    public RouterFunction<ServerResponse> route(TaskHandler taskHandler)
```

Comment:
This method is not annotated with @Bean annotation. Thus, not a valid router function. Please correct.